package.json

```diff
@@ -11,27 +11,28 @@
     "autoprefixer": "^6.5.3",
     "awesome-phonenumber": "^1.1.3",
     "babel": "6.5.2",
-    "babel-core": "^6.18.2",
-    "babel-eslint": "^6.1.0",
-    "babel-loader": "^6.2.10",
-    "babel-plugin-jsx": "1.2.0",
+    "babel-core": "latest",
```

Comment:
I suppose so, I just did `yarn add package@latest`, I didn't realize it hadn't set the version number here. I'll update it so that `package.json` has the right version number.